pub_key.go

```diff
@@ -76,7 +149,40 @@ func (pubKey PubKeyEd25519) ToCurve25519() *[32]byte {
 }
 
 func (pubKey PubKeyEd25519) String() string {
-	return fmt.Sprintf("PubKeyEd25519{%X}", pubKey[:])
+	marshaled, err := cdc.MarshalBinary(pubKey)
+	if err != nil {
+		return err.Error()
+	}
+	conv, err := bech32cosmos.ConvertBits(marshaled, 8, 5, true)
+
+	bech, err := bech32cosmos.Encode("CSMSPUB", conv)
```

Comment:
lets use constants instead of the magic strings for things we don't give external control over.

Is that right? Addresses will be `CSMSACC` or  `CSMSVAL` but pubkeys will always be `CSMSPUB` ?